src/Instrumentation/Laravel/src/HttpInstrumentation.php

```diff
@@ -79,6 +80,15 @@ public static function register(CachedInstrumentation $instrumentation): void
                     $span->setAttribute(TraceAttributes::NETWORK_PROTOCOL_VERSION, $response->getProtocolVersion());
                     $span->setAttribute(TraceAttributes::HTTP_RESPONSE_BODY_SIZE, $response->headers->get('Content-Length'));
                 }
+                if(($route = Route::getCurrentRoute()?->uri()) !== null) {
+                    $request = ($params[0] instanceof Request) ? $params[0] : null;
+
+                    if (! str_starts_with($route, '/')) {
```

Comment:
what happens here if it's a [named route](https://laravel.com/docs/10.x/routing#named-routes) rather than a uri pattern?

Reply:
Sorry, what I was trying to say was "should the route name be used as the span name if one is defined for a route" ? https://opentelemetry.io/docs/specs/semconv/http/http-spans/#name doesn't go down to that level of detail, and I don't think we're consistent between various frameworks' auto-instrumentation, so I don't have a strong opinion here.

Reply:
Ah, I understand now.

That is actually a good question, which I also don't have a strong opinion of.

One argument to use URI instead of route name would be that it always exists for registered routes, and thus more consistent.

Reply:
no problem, let's leave it as is for now :)